src/app/Task.ts

```diff
@@ -0,0 +1,8 @@
+export interface Task {
+  name: string;
+  id: number;
+  done: boolean;
+  edit: boolean;
+  trash: boolean;
+  startDate: number;
+}
```

Comment:
Instead of creating seperate file for interfaces, we could introduce `*.d.ts` type files which will store type definations.

Reply:
This interface seems to represent the state of the `TaskCard` rather than the `Task`

Reply:
this interface is not needed anymore. so I've deleted Task.ts file